OrbitGl/TimeGraph.cpp

```diff
@@ -592,15 +592,73 @@ void TimeGraph::Draw(GlCanvas* canvas, bool a_Picking) {
   m_NeedsRedraw = false;
 }
 
+namespace {
+
+std::pair<std::string, std::string> GetLabelBetweenIterators(
+    const TextBox* box_a, const TextBox* box_b) {
+  double micros = MicroSecondsFromTicks(box_a->GetTimer().m_Start,
+                                        box_b->GetTimer().m_Start);
+
+  std::string time = GetPrettyTime(micros * 0.001);
+  std::string function_from =
+      Capture::GAddressToFunctionName[box_a->GetTimer().m_FunctionAddress];
+  std::string function_to =
+      Capture::GAddressToFunctionName[box_b->GetTimer().m_FunctionAddress];
+  return std::make_pair(absl::StrFormat("%s to %s", function_from, function_to),
+                        time);
+}
+
+Color GetIteratorBoxColor(uint64_t index) {
+  constexpr uint64_t kNumColors = 4;
+  Color colors[kNumColors] = {Color(73, 208, 242, 80), Color(0, 90, 112, 80),
+                              Color(0, 192, 240, 80), Color(40, 103, 122, 80)};
```

Comment:
Shall we give names to these colours (kLightBlue1, kDarkBlue1 (or kGrey1), kLightBlue2, ...)?

Reply:
That's a good idea, done.

Reply:
Also reduced the number of colors to just two, because there wasn't really a large difference between 0, 2 and 1, 3.